main/file/APKBUILD

```diff
@@ -4,13 +4,21 @@ pkgname=file
 pkgver=5.36
 pkgrel=0
 pkgdesc="File type identification utility"
-url="http://www.darwinsys.com/file"
+url="https://www.darwinsys.com/file"
```

Comment:
with this change we need bump pkgrel, because `apk info --webpage` will give different result.

Reply:
Isn't this updated anyway with the next change?

Reply:
btw: would be great if you could give a feedback here:
https://github.com/alpinelinux/aports/pull/4516#issuecomment-429306674